packages/ethers-cf-purelocal/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ethers-cf-purelocal",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "tsc && node test.js"
```

Comment:
Useful to split up the build and test steps. Also, we can use `jest` to run the tests (which will read `.ts` files)